Trell/CliCommands/RunCommand.cs

```diff
@@ -18,38 +21,122 @@ public class RunCommandSettings : CommandSettings {
     [CommandOption("--config")]
     public string? Config { get; set; }
 
-    [CommandOption("--log-level")]
-    public LogEventLevel? LogLevel { get; set; } = null;
+    [CommandOption("-u|--user-id <user-id>"), DefaultValue("new_user"), Description("User id to send in work order; also used for worker id resolution")]
```

Comment:
```suggestion
    [CommandOption("-u|--user <username>"), DefaultValue("new_user"), Description("User to send in work order")]
```

And make `trell init` support `-u|--user <username>`.

Reply:
Resolved in both RunCommand and InitCommand in commit 0caa77c